### ISSUE TITLE
Documenting all the Analysis you did for this Project in the ReadMe

### ISSUE THREAD

**Issue by carlos-collado** [epfl-ada/ada-2023-project-amonavis#14]
Whatever your task for this milestone is, they want us to document it well in the readme, so create a title with what you did and explain the motivation, methods, results, and insights of the analysis.

**Comment by DanielX278** [comment]
yes guys, and be careful to the length of the doc! I tried to effectively explain all the rationale behind the main project and analyses, and we are already at over 800 words. feel free to edit, delete and add whatever comes to your mind, both to the methods part and to the others. 

that said, pls check the parts in italic, where I am the least sure.
